tests/unit/test_is_read_message.py

```diff
@@ -95,3 +96,24 @@ def test_not_read_zmqlet():
         d.tags['id'] = 2
         msg = Message(None, req, 'tmp', '')
         z.send_message(msg)
+
+
+def test_recv_message_zmqlet():
+    zmqlet1 = Zmqlet(args1, default_logger)
+    zmqlet2 = Zmqlet(args2, default_logger)
+    req = jina_pb2.RequestProto()
+    req.request_id = random_identity()
+    doc = req.index.docs.add()
+    doc.tags['id'] = 2
+    msg = Message(None, req, 'tmp', '')
+    called = False
+
+    def callback(msg_):
+        nonlocal called
+        called = True
```

Comment:
use mocker as in many integration tests to ensure callback is called

Reply:
Thanks, addressed.